Copyright (C) 2008 John R. Rose.  This package may be used for any
purpose, provided that this copyright notice is preserved intact.

THIS PACKAGE IS PROVIDED "AS IS" AND WITHOUT ANY EXPRESS OR
IMPLIED WARRANTIES, INCLUDING, WITHOUT LIMITATION, THE IMPLIED
WARRANTIES OF MERCHANTIBILITY AND FITNESS FOR A PARTICULAR PURPOSE.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -32,6 +32,7 @@
 Makefile
 libdisasm/Makefile
 x86dis/Makefile
+hsdis/Makefile
 test/Makefile
 doc/Makefile
 man/Makefile
